### DISCUSSION TITLE
Staking not available

### DISCUSSION THREAD

**Discussion by Tayeb234** [iotaledger/firefly#1848] [Help & Support]
Staking is not available in my newly updated Firefly wallet, it reflects the new links to assembly and Shimmer sites, however it says that staking is not available and nodes need to be changed - when changed this does not fix the issue.

**Comment by r-c-k** [comment]
Please try adding `https://mainnet-node.tanglebay.com/` to the node list for now (Settings - Network config - Manual - Add node). Set it as primary and then try again.

Alternatively join the [IOTA Discord](https://discord.iota.org) and contact me there (rck#3810).

**Reply by diewars** [reply]
Worked immediately, many thanks!
